### ISSUE TITLE
npm warnings when installing

### ISSUE THREAD

**Issue by yuhui** [Adobe-Marketing-Cloud/reactor-uploader#39]
When running `npx @adobe/reactor-uploader` for the first time, a list of warnings about deprecated dependencies are listed.

### Expected Behaviour
No warnings are listed.

### Actual Behaviour
Warnings appear.

### Reproduce Scenario (including but not limited to)

macOS Big Sur 11.0.1
node 15.2.1
npm 7.0.8

`node` and `npm` installed with `brew`.

#### Steps to Reproduce

1. Run `npx @adobe/reactor-uploader`

#### Sample Code that illustrates the problem

```
yuhui@computer launch-extension-folder % npx @adobe/reactor-uploader ...
Need to install the following packages:
  @adobe/reactor-uploader
Ok to proceed? (y) y
npm WARN deprecated request-promise-native@1.0.9: request-promise-native has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
... [logging about extension package]
✔ The extension package was successfully processed.
```

**Comment by brenthosie** [comment]
All of these dependencies are a part of the deprecation of `request-promise-native` and is tracked by DTM-15991.

**Comment by yuhui** [comment]
Updated set of warnings

Environment:
node 16.1.0
npm 7.11.2

```
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated request-promise-native@1.0.9: request-promise-native has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
```

**Comment by thebenrob** [comment]
@adobe export issue to Jira project PDCL

**Comment by thebenrob** [comment]
@adobe export issue to Jira project PDCL

**Comment by brenthosie** [comment]
Closed by #48
